src/avram/queryable.cr

```diff
@@ -245,9 +251,19 @@ module Avram::Queryable(T)
     @preloads << block
   end
 
+  def cache_store
+    Fiber.current.query_cache
+  end
+
+  def cache_key : String
+    [query.statement, query.args].join(':')
```

Comment:
I guess if you wanted a different cache_key, you could just override this method in your query class 🤔